client/src/App.scss

```diff
@@ -2,3 +2,7 @@
   text-align: center;
 }
 
+body {
+  overflow-x: hidden;
+}
```

Comment:
I think there's an... index.css where the body tag is used that this might be added to  in a later PR

Reply:
Yeah, either way. Both index.css and App.scss are very small and could be combined. index.css is imported in index.js and App.scss is imported in App.jsx.

They could definitely be combined into one file.